datasets/SCCS/convert.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf8
+from __future__ import unicode_literals, print_function, division
+from collections import Counter, defaultdict, OrderedDict
+
+import attr
+from clldutils.path import Path, read_text, write_text
+from clldutils.dsv import reader, UnicodeWriter
+from clldutils.misc import nfilter
+from pyglottolog.references import BibFile
+
+from pydplace.api import Society
+
+#
+# read wnai.bib
+# write mapping of bibkeys to bib.json
+# read sources.bib
+# merge records from wnai.bib not yet in sources.bib
+# write sources.bib
+#
```

Comment:
Dead comments? or TODOs?

Reply:
The whole file is only temporary, until I get the final set of data from @kirbykat

Reply:
👍